docs/build_arm.md

```diff
@@ -17,9 +17,10 @@ following order:
    install Bazel, see the section
    [Cross-compiling LCE with Make](#cross-compiling-lce-with-make).
 
-In this guide, we use the [LCE benchmark tool](../larq_compute_engine/tflite/benchmark)
-source to build an inference binary. See [here](./inference.md) to find out how
-you can create your own custom LCE inference application.
+This guide will show you how to build the [LCE example application](../examples/lce_minimal.cc)
+and the [LCE benchmark tool](../larq_compute_engine/tflite/benchmark).
```

Comment:
I don't think we need to give two examples here. just showing  example LCE application is enough. Building LCE benchmark tool does not add any new values or insight.